lib/modules/manager/npm/post-update/types.ts

```diff
@@ -31,7 +31,8 @@ export interface GenerateLockFileResult {
 }
 
 export interface PnpmLockFile {
-  lockfileVersion?: number;
+  lockfileVersion: number | string;
+  packages?: Record<string, any>;
```

Comment:
```suggestion
  packages?: Record<string, unknown>;
```

We don't care, so we don't like to have a silent any here. 😉